index.js

```diff
@@ -49,6 +50,13 @@ export async function setupRuby(options) {
 
   setupPath(newPathEntries)
 
+  // When setup-ruby is used by other actions, this allows code in them to run
+  // before 'bundle install'.  Installed dependencies may require additional
+  // libraries & headers, build tools, etc.
+  if (inputs['func'] instanceof Function) {
```

Comment:
`func` is rather unspecific, maybe `afterSetupPathHook` ?

Reply:
Done